app/controllers/talks_controller.rb

```diff
@@ -38,7 +38,18 @@ def destroy
 
   def upvote
```

Comment:
Anyway you can figure out how to put the `vote` method in the model? That would be the best, the controller is getting a little bloated. As a best practice we should figure out how to keep our controller skinny.